### DISCUSSION TITLE
Mobile Menu Installation

### DISCUSSION THREAD

**Discussion by jpgarza93** [jothepro/doxygen-awesome-css#74] [General]
I am not seeing the mobile menu on my site: https://rapidcode.roboticsys.com/

Is there anything in particular that I have to do to activate it?

**Note**: I have tried building with Doxygen v1.8.20 and v1.9.4

**Comment by jothepro** [comment]
Hm, that's weird. In v1.8.20 it wasn't available, but in v1.9.4 the hamburger menu should be present. Maybe this is because you defined some custom menu entries? Possibly the hamburger menu only works for the default entries (MainPage, Related Pages, Namespaces, Classes, Files...)? It's just a wild guess though, I haven't validated it.

But to be clear: The hamburger menu is not a feature added by *doxygen-awesome-css*, it's a default doxygen thing. So you may want to ask in [doxygen/doxygen](https://github.com/doxygen/doxygen) how to activate it for custom menu items. 🤔

**Reply by jpgarza93** [reply]
Sounds good, thank you
